src/components/user-profile.component.js

```diff
@@ -43,7 +43,7 @@ export const UserProfile = ({
           })}
       >
         <Text style={styles.unitNumber}>
-          {user.public_repos}
+          {user.public_repos + (user.total_private_repos || 0)}
```

Comment:
Just want to ask - is `total_private_repos` undefined when a user has no private repos? Or does it default to 0?